index.es6.js

```diff
@@ -0,0 +1,305 @@
+'use strict';
+
+var _ = require('lodash');
+var axios = require('axios');
+var resolveResponse = require('contentful-resolve-response');
+var querystring = require('querystring');
```

Comment:
Since we are going ES6 here, we might want to use the [`import`](https://babeljs.io/docs/learn-es6/#modules) statement.